app/models/i14y_search.rb

```diff
@@ -2,12 +2,14 @@ class I14ySearch < FilterableSearch
   include SearchInitializer
   include Govboxable
   I14Y_SUCCESS = 200
-  attr_reader :collection
+  attr_reader :collection,
+              :dc
```

Comment:
@MothOnMars does the i14y search interface support the `dc` attribute? I'm concerned that @viclim might be sending an attribute that will be silently dropped on the floor without the expected behavior.

Reply:
@nickmarden I think you're right about `dc` being a useless attribute. [As shown here](https://github.com/GSA/search-gov/blob/master/app/models/i14y_search.rb#L53), all the affiliate's handles are being sent to the i14y service.

So, what's the relationship between a DocumentCollection and an I14yDrawer?

Reply:
@viclim a `DocumentCollection` is an override for the default search scope set in the domains list. The folders specified in a `DocumentCollection` may or may not be held in an i14y drawer, they may be over in the SearchGov index.

The `DocumentCollection` id can be sent in as a param, to tell the system which override to use....

`search.usa.gov/search?affiliate=foo&query=bar` -> this uses the Domains list as the scope for the query 
`search.usa.gov/search/docs?affiliate=foo&dc=12345&query=bar` -> this uses the list of URL path prefixes listed in the `DocumentCollection 12345`

In either of these cases, the source index may be 
`BingV6` (if `search_engine=BingV6`) 
OR 
`SearchGov + i14y Drawer(s)` (if `search_engine=SearchGov` AND `gets_i14y_results=true`) 
OR 
`SearchGov` (if `search_engine=SearchGov` AND `gets_i14y_results=false`) 
OR 
`i14y Drawer(s)` (if `search_engine=BingV6` AND `gets_i14y_results=true`)

SearchGov is itself an i14y drawer, but related to the affiliate as a search engine, not as an i14y drawer.